app/assets/stylesheets/application.scss

```diff
@@ -33,31 +33,73 @@ $navbar-brand-padding-y: 8px;
 
 @import "bootstrap";
 
-.navbar-brand img {height: 84px}
-.nav-icon img {width: 50px}
-.nav-icon {height: 100px; width: 100px; font-size: 12px}
-.nav-link {text-align: center}
-.background-purple {background-color: $bw-color-purple}
-.background-dark-purple {background-color: $bw-color-dark-purple}
-.background-slate {background-color: $bw-color-slate}
-.background-blue {background-color: $bw-color-blue}
-.background-darkest-purple {background-color: $bw-color-darkest-purple}
-.navbar-text .pull-right {position: fixed}
+.navbar-brand img {
+  height: 84px;
+}
+// .nav-icon img {
+//   width: 5em;
+// }
+.nav-icon {
+  height: 100px;
+  width: 100px;
+  font-size: 12px;
+}
+.nav-link {
+  text-align: center;
+}
+.background-purple {
+  background-color: $bw-color-purple;
+}
+.background-dark-purple {
+  background-color: $bw-color-dark-purple;
+}
+.background-slate {
+  background-color: $bw-color-slate;
+}
+.background-blue {
+  background-color: $bw-color-blue;
+}
+.background-darkest-purple {
+  background-color: $bw-color-darkest-purple;
+}
+.navbar-text .pull-right {
+  position: fixed;
+}
+
+.nav-item {
+  display: flex;
+  align-items: center;
+}
 
+#nav-first img {
```

Comment:
Not sure that using `ids` inside the css is a good approach, personally I would keep `ids` outside the css